src/domain/image/data/retrieve.ts

```diff
@@ -1,7 +1,12 @@
 
-import ImageData from './ImageData';
+import database from '../../../integrations/database/module';
+import type ImageData from './ImageData';
+import { RECORD_TYPE } from './constants.js';
+import createData from './createData';
 
-export default async function retrieve(id: string): Promise<ImageData>
+export default async function retrieve(imageId: string): Promise<ImageData>
```

Comment:
The variable `id` is used in all cases for retrieve.

Reply:
Changed to id.